src/umbraco.cms/Actions/ActionExportMember.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using umbraco.BasePages;
+using umbraco.interfaces;
+
+namespace umbraco.cms.Actions
+{
+    public class ActionExportMember: IAction
```

Comment:
We don't need an IAction for this, IAction is only relevant for Content items (also if we were to use this it would need to be internal since we don't want to expose legacy code publicly)